gatsby-node.js

```diff
@@ -39,10 +40,10 @@ exports.createPages = ({ graphql, actions }) => {
           reject(result.errors)
         }
 
-        // Create blog posts pages.
+        const category = result.data.allMarkdownRemark.group
         const posts = result.data.allMarkdownRemark.edges
 
-        result.data.allMarkdownRemark.group.forEach(cate => {
+        category.forEach(cate => {
```

Comment:
Oh i had this same question but then deleted -- I think the way it works is it's iterating through different node types, because further down there's something similar. Will let @jiahaodavid comment 😄

Reply:
Yes. I am too bad on the naming. It looks weird to me too. I found out I always name thing very randomly. I will pay more attention to naming staff. 🤔